main.css

```diff
@@ -0,0 +1,164 @@
+body {
+  font: 16px/1.2 Source Sans Pro, sans-serif;
+  padding: 0;
+  margin: 0;
+  min-width: 320px;
+  background: #fff4f0;
+}
+
+.clearfix::after {
+  content: "";
+  display: table;
+  clear: both;
+}
+
+.wrapper {
+  position: relative;
+}
+
+.content {
+  max-width: 1000px;
+  margin: 0 auto;
+  padding: 0 10px;
+}
+
+.header {
+  padding: 0 15%; 
+  margin: 0 
+  auto;
+  background-color: white;
+  -webkit-box-shadow: 0px 4px 5px -2px rgba(0,0,0,0.75);
+  -moz-box-shadow: 0px 4px 5px -2px rgba(0,0,0,0.75);
+  box-shadow: 0px 4px 5px -2px rgba(0,0,0,0.75);
+}
+
+.header__nav-list {
+  float: right;
+}
+
+.header__nav-item {
+  display: inline;
+  margin-left: 15px;
+}
+
+.header__nav-item:hover {
+  color: orange;
+}
+
+.header__nav-item>a:hover {
+  color: orange;
+}
+
+.header__sign-in-icon {
+  position: relative;
+  top: 3px;
+  width: 18px;
+}
+
+
+
+.header__burger-icon {
+  position: relative;
+  display: none;
+  float: left;
+  padding: 5px;
+  top: 20px;
+  left: 10px;
+  margin-bottom: 25px;
```

Comment:
Please remove this margin from here. if you need some space inside header use padding for header.